### ISSUE TITLE
Add support for switch statements in padded-blocks

### ISSUE THREAD

**Issue by alberto** [eslint/eslint#5056]
This should be a problem:
```
/*eslint padded-blocks: [2, "always"] */

switch (a) {
  case 0: foo();
}
```

**Comment by nzakas** [comment]
A problem by default or a problem with an option?

**Comment by alberto** [comment]
By default, IMO. Do you see any reason why it should be different from other blocks? 
We already include them in `block-spacing` too.

**Comment by nzakas** [comment]
Just trying to think of any reason we might have omitted this initially, but can't come up with anything. I do wonder if people want to style `switch` separately, but I guess we will find that out.

**Comment by lo1tuma** [comment]
I guess then we should also apply this rule for `ClassBody`.

**Comment by alberto** [comment]
@lo1tuma Yeah I looked into that. The `ClassBody` body is already a `BlockStatement`. Same with parenthesized arrow function body.

**Comment by lo1tuma** [comment]
@alberto That's not a correct a `ClassBody` node is from type `ClassBody`.

```js
/*eslint padded-blocks: [2, "always"] */

class Foo {
    foo() {}
}
```

This doesn't report a problem in the online demo.

**Comment by alberto** [comment]
Oops, you are right. (I meant the `body` of `ClassBody`, but it's not a `BlockStatement` either)

**Comment by btmills** [comment]
I just made a [proposal](https://github.com/eslint/eslint/issues/5092#issuecomment-177662072) on #5092 that would add support for class bodies. I won't cross-post the entire proposal here, but tl;dr:

```json
{
    "rules": {
        "padded-blocks": [2, {
            "blocks": "never",
            "classes": "always",
            "switches": "never"
        }]
    }
}
```

That's if you all think switches should be separate from blocks. If not, then that proposal is irrelevant.

**Comment by alberto** [comment]
Per the conversation in #5092, this would also be breaking, so the same decision should be applied here.
